### PR TITLE
Adding assignment

### PR DESCRIPTION
**Pull request by Gagan2095**
Created an ERC-20 token using the hardhat framework. The contract code is written in the Contract folder, all the standards are applied to it. I moved the ignition folder in the gitignore file, and also added the README file in the branch.
There are some configurations for deployment and verification purposes which are in hardhat.config.js file. I have declared three environment variables locally in the project which are being used in the hardhat config file. Applied the standard solidity style guide and Natspec on every part of code.

### CODE REVIEW

**Comment by MoFahimKh** [comment]
Please fix the PR description, it does not justifies the work done in the PR.

File: test/Token.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { expect } = require("chai");
+
+describe("Coins", function () {
```
Comment (MoFahimKh) [inline test/Token.js:3]: write tests for all the written functions in the contract.

**Comment by MoFahimKh** [comment]
I am unable to find a readme file in the PR.

**Review by MoFahimKh — CHANGES_REQUESTED** [review]
Is the ignition file required to be pushed in the repo? if not pleas remove

**Comment by Gagan2095** [comment]
> I am unable to find a readme file in the PR.

Because I first pushed the README file on main branch

**Comment by MoFahimKh** [comment]
> does

add it here, will review your readme as well

**Comment by Gagan2095** [comment]
> Please fix the PR description, it does not justifies the work done in the PR.

ok

**Comment by Gagan2095** [comment]
> Is the ignition file required to be pushed in the repo? if not pleas remove

ok

**Comment by Gagan2095** [comment]
> Please fix the PR description, it does not justifies the work done in the PR.

fixed.

**Comment by Gagan2095** [comment]
> I am unable to find a readme file in the PR.

Please check now.

**Pull request merged**